pkg/api/vlabs/types.go

```diff
@@ -883,6 +885,14 @@ func (l *LinuxProfile) HasCustomNodesDNS() bool {
 	return false
 }
 
+// IsCsiProxyEnabled returns true if csi proxy service should be enable for Windows nodes
+func (w *WindowsProfile) IsCsiProxyEnabled() bool {
```

Comment:
ditto ditto

Reply:
Updating everywhere (except in the powershell code :))